### PR TITLE
Add disk cache option

### PR DESCRIPTION
**Pull request by katcipis**

### CODE REVIEW

File: examples/azure/backup/build.sh [hunk 1]
```diff
@@ -107,7 +107,7 @@ range     <= split($sequence, "\n")
 print("creating %q disks with size %q\n", $vm_disks_count, $vm_disks_size)
 
 for i in $range {
-	azure_vm_disk_attach_new($vm_name, $group, "disk" + $i, $vm_disks_size, "Premium_LRS")
+	azure_vm_disk_attach_new($vm_name, $group, "disk" + $i, $vm_disks_size, "Premium_LRS", "None")
 }
```
Comment (kamilash) [inline examples/azure/backup/build.sh]: Why none here? This will be the default configuration?

Comment (katcipis) [reply examples/azure/backup/build.sh]: No, it is just an example and I was not interested on showing different configurations for this on the example right now =P

Comment (kamilash) [reply examples/azure/backup/build.sh]: Understood :)

Comment (katcipis) [reply examples/azure/backup/build.sh]: But it would be cool to have examples for this too ;-)

**Pull request merged**